content/blog/monitor-application-performance-across-hybrid-cloud-environment-using-apache-skywalking-and-service-mesh.md

```diff
@@ -0,0 +1,333 @@
+---
+title: "Addressing hybrid cloud application challenges using HPE GreenLake for
+  Private Cloud Enterprise – Part 2: Application monitoring"
+date: 2023-01-05T06:42:26.899Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+disable: false
+tags:
+  - hpe-greenlake, hpe-greenlake-for-private-cloud-enterprise, Apache
+    SkyWalking, service mesh, istio, application performance monitoring
+---
+## Introduction
+
+In [my previous blog post](https://developer.hpe.com/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise/), I covered the detailed process of deploying the complex Online Boutique application in a hybrid cloud environment, across the public EKS cluster from AWS to the private Kubernetes cluster in [HPE GreenLake for Private Cloud Enterprise](https://www.hpe.com/us/en/greenlake/private-cloud-enterprise.html). This hybrid cloud model amplifies the benefits of both private and public clouds and allows for more seamless integration across technical barriers. It enables the enterprise to rely on the security of on-premise data centers while taking advantage of the agility of managing the front-end of an application in the public cloud. This model is becoming increasingly popular as more businesses and enterprises shift toward cloud-based computing. 
```

Comment:
This may have been my fault, as I may have suggested an edit that I shouldn't have, but I went back and double-checked.... "on-premise" should be "on-premises", so where it says "It enables the enterprise to rely on the security of on-premise data centers..." you might want to change that (back?) to "on-premises.